### PR TITLE
update kusama fees to v1.0.0 and rollback polkadot links to v0.9.43

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR updates the fees page for Kusama v1.0.0 and reverts the Polkadot links back to the deprecated repo where v0.9.43 was released. As announced in the [v1.0.0 release notes](https://github.com/paritytech/polkadot/releases/tag/v1.0.0), all runtime code has been moved to the fellowship PR, so this impacts the benchmarked weights.

To summarize:
- For v0.9.43 we should be using the now-deprecated repo for Polkadot weights: https://github.com/paritytech/polkadot/blob/v0.9.43
- For v1.0.0, we should be using the fellowship repo for Kusama weights: https://github.com/polkadot-fellows/runtimes/tree/v1.0.0

Once Polkadot is on v1.0.0, we'll have to update the links to point to the fellowship PR.

This PR also updates the polkadot-sdk links on the fees page so that we point to v1.1.0, which is the version we migrated to as of RT2601.

You can double-check the calculated fees by looking at any xcKSM > KSM transaction that includes the instructions listed in the docs (`WithdrawAsset`, `ClearOrigin`, `BuyExecution`, and `DepositAsset`). For example:

- XCM Transfer: https://moonriver.subscan.io/xcm_message/kusama-4cb563c7c97599c510f6817530ae188894f20961
- Extrinsic on Kusama: https://kusama.subscan.io/extrinsic/20567957-0?event=20567957-3
- Block that includes the Extrinsic: https://kusama.subscan.io/block/20567957?tab=event
- View the `balances(Deposit)` events (should be two per XCM) and the `balances(Withdraw)` event (should be one per XCM):
   - [`balances(Withdraw)`](https://kusama.subscan.io/block/20567957?tab=event&&event=20567957-0) - 13.0000900058 KSM 👈 Transfer amount
   - [`balances(Deposit)`](https://kusama.subscan.io/block/20567957?tab=event&&event=20567957-1) - 13.000008226255 KSM 👈 Amount received in destination account (less fees)
   - [`balances(Deposit)`](https://kusama.subscan.io/block/20567957?tab=event&&event=20567957-2) - 0.000081779545 KSM 👈 Fees paid for the XCM execution
   
Goes with CN PR: https://github.com/moonbeam-foundation/moonbeam-docs-cn/pull/360

### CODE REVIEW

**Comment by eshaben** [comment]
Some of this overlaps with what I've done in the XCM overhaul. This PR just gets everything back to working order and then everything will be cleaned up in the XCM overhaul PR

File: builders/interoperability/xcm/fees.md [hunk 1]
```diff
@@ -114,26 +132,26 @@ As an example, you can calculate the total cost of DOT for sending an XCM messag
...
 
 With the instruction weight cost established, you can calculate the cost of the instruction in KSM.
 
-In Kusama, the [`ExtrinsicBaseWeight`](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/kusama/constants/src/weights/extrinsic_weights.rs#L56){target=_blank} is set to `{{ networks.kusama.extrinsic_base_weight.display }}` which is [mapped to 1/10th](https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/runtime/kusama/constants/src/lib.rs#L87){target=_blank} of a cent. Where 1 cent is `10^12 / 30,000`.
+In Kusama, the [`ExtrinsicBaseWeight`](https://github.com/polkadot-fellows/runtimes/blob/{{networks.polkadot.spec_version}}/relay/kusama/constants/src/weights/extrinsic_weights.rs#L56){target=_blank} is set to `{{ networks.kusama.extrinsic_base_weight.display }}` which is [mapped to 1/10th](https://github.com/polkadot-fellows/runtimes/blob/{{networks.kusama.spec_version}}/relay/kusama/constants/src/lib.rs#L87){target=_blank} of a cent. Where 1 cent is `10^12 / 30,000`.
```
Comment (themacexpert) [inline builders/interoperability/xcm/fees.md:154]: 1st Link 404s

**Review by themacexpert — APPROVED** [review]
LGTM

File: builders/interoperability/xcm/fees.md (hunk 1 above)
Comment (eshaben) [reply builders/interoperability/xcm/fees.md]: Thanks!

**Pull request merged**